docs/en/operations/settings/merge-tree-settings.md

```diff
@@ -106,21 +106,20 @@ Possible values:
 Default value: 1.
 
 The delay (in milliseconds) for `INSERT` is calculated by the formula:
-
 ```code
 max_k = parts_to_throw_insert - parts_to_delay_insert
 k = 1 + parts_count_in_partition - parts_to_delay_insert
 delay_milliseconds = pow(max_delay_to_insert * 1000, k / max_k)
 ```
+For example, if a partition has 299 active parts and parts_to_throw_insert = 300, parts_to_delay_insert = 150, max_delay_to_insert = 1, `INSERT` is delayed for `pow( 1 * 1000, (1 + 299 - 150) / (300 - 150) ) = 1000` milliseconds.
 
 Starting from version 23.1 formula has been changed to:
 ```code
-allowed_parts_over_threshold = parts_to_throw_insert - parts_to_delay_insert + 1
+allowed_parts_over_threshold = parts_to_throw_insert - parts_to_delay_insert
 parts_over_threshold = parts_count_in_partition - parts_to_delay_insert + 1
 delay_milliseconds = max(min_delay_to_insert_ms, (max_delay_to_insert * 1000) * parts_over_threshold / allowed_parts_over_threshold)
 ```
-
-For example if a partition has 299 active parts and parts_to_throw_insert = 300, parts_to_delay_insert = 150, max_delay_to_insert = 1, `INSERT` is delayed for `pow( 1 * 1000, (1 + 299 - 150) / (300 - 150) ) = 1000` milliseconds.
+For example, if a partition has 224 active parts and parts_to_throw_insert = 300, parts_to_delay_insert = 150, max_delay_to_insert = 1, min_delay_to_insert_ms = 10, `INSERT` is delayed for `max( 10, 1 * 1000 * (224 - 150 + 1) / (300 - 150) ) = 500` milliseconds.
```

Comment:
Why do we use a different number in the examples? Examples with the same number of active parts will be easier to compare what was changed

Reply:
If you use the same numbers as in existed example the delay will be the same. So, there is no point to compare 1000 milliseconds with 1000 milliseconds. Probably, to show difference we can use number from new example in old one, - in this case the delay will be different